cql3/selection/selectable.cc

```diff
@@ -104,6 +104,16 @@ make_count_rows_function_expression() {
                     std::vector<cql3::expr::expression>()};
 }
 
+expr::expression
+make_function_expression(const ::shared_ptr<functions::function>& function, const std::vector<sstring>& args) {
```

Comment:
Why sstring parameters and not expression parameters?

Reply:
And it's a function call expression, not a function expression. I don't see why you can't use expr::function_call directly.